zeps/draft/ZEP0001.md

```diff
@@ -22,374 +24,102 @@ Created: 2022-05-23
-the zarr-specs website after a review process by the Zarr community,
-thus providing a common point of discovery and coordination across the
-community. However, we do not want to obstruct individuals or
-organisation who need to innovate or experiment or who have
-specialised needs which are better served by managing and publishing
-specifications in a different way.
+@@TODO
 
-Note that we also anticipate that user groups from different domains
-may wish to publish **usage convention specifications**, by which we
-mean specifications that define conventions for the use of Zarr for a
-particular type of data, such as restrictions and expectations
-regarding the groups, arrays and attributes that will be found within
-a Zarr dataset. How these usage convention specifications will be
-managed, published and used is out of scope for this ZEP.
+This section should list relevant and/or similar technologies, possibly in other
+libraries. It does not need to be comprehensive, just list the major examples
+of prior and relevant art.
 
-A draft of the Zarr version 3 core specification is available at the
-following URL:
+## Implementation
 
-   * https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html
+@@TODO
 
-Note publication of the draft specification on the zarr-specs website
-preceded establishiment of the ZEP process, and does not imply
-acceptance of this ZEP. The specification should be considered a
-proposal at this time, pending acceptance of this ZEP.
+This section lists the major steps required to implement the ZEP. Where
+possible, it should be noted where one step is dependent on another, and which
+steps may be optionally omitted. Where it makes sense, each step should include
+a link to related pull requests as the implementation progresses.
 
-To illustrate the principle of the other specification types, the
-following specifications have also been published on the zarr-specs
-website:
+Any pull requests or development branches containing work on this ZEP be linked
+to from here. (A ZEP does not need to be implemented in a single pull request
+if it makes sense to implement it in discrete phases).
 
-* [Codecs](https://zarr-specs.readthedocs.io/en/latest/codecs.html)
+## Alternatives
 
-* Stores - [File system
-  store](https://zarr-specs.readthedocs.io/en/latest/stores/filesystem/v1.0.html)
+@@TODO
 
-* Array extensions -
-  [Filters](https://zarr-specs.readthedocs.io/en/latest/extensions/array-extensions/filters/v1.0.html)
+If there were any alternative solutions to solving the same problem, they should
+be discussed here, along with a justification for the chosen approach.
 
-* Data types - [Datetime data
-  types](https://zarr-specs.readthedocs.io/en/latest/extensions/data-types/datetime/v1.0.html)
+## Discussion
 
-* Storage transformers - [Sharding storage
-  transformer](https://zarr-specs.readthedocs.io/en/latest/extensions/storage-transformers/sharding/v1.0.html)
+@@TODO
```

Comment:
Is this a reversion to a previous version?

Reply:
Ah, I think I convinced myself that the reverted version matches what's currently merged and that the previous changes are what's at #1, right?

Reply:
> Is this a reversion to a previous version?

Yes.

I've reverted the ZEP0001.md to default text because there's still work needed to be done by Jonathon Striebel. Once it's done I'll push the complete ZEP0001.md to the webpage.